CONTRIBUTING.adoc

```diff
@@ -9,7 +9,7 @@
 
 The guides toolchain is heavily dependent on AsciiDoc and Node.
 
-. Install your favourite tool for editing AsciiDoc (for example, Atom).
+. Install your favourite tool for editing AsciiDoc (for example, Atom or VS Code).
```

Comment:
```suggestion
. Install your favourite tool for editing AsciiDoc.
```

Rather than worry about what is the correct product name (should it be "VS Code" or "Visual Studio Code", and should we include the "Microsoft" prefix), and to avoid potential accusations of favouring one tool over another (we can't name them all), perhaps don't name any tool explicitly?